### DISCUSSION TITLE
Significant usability regressions in v0.85 on Mac

### DISCUSSION THREAD

**Discussion by ThatAdamGuy** [athensresearch/athens#1267] [Archive]
Apologies in advance for not offering detailed bugs tonight due to being pretty time-strapped.

On an M1 Mac Mini running latest Mac Big Sur OS, installed Athens Mac app this morning.

- Block selection is consistently off when selecting a significant chunk via shift and arrow keys (always overshoots)
- Cursor very often disappears
- At least a couple times, was unable to select anything and had to close and restart the app
- Deleted blocks sometimes re-appear

**Comment by jefftangx** [comment]
I'm noticing the freezing on click too on Linux, but not sure if I understand your other problems. Is it possible to send a video recording of your issues @ThatAdamGuy ? X-post from #1265

**Reply by jefftangx** [reply]
Please update to beta.86 and let us know if any issues still exist.

**Comment by jsmorabito** [comment]
@ThatAdamGuy are you still experiencing this issue after the update?

**Comment by ThatAdamGuy** [comment]
I'm sorry, but to be frank, I have temporarily stepped away from using
Athens, since I need something a bit more stable for my day-to-day work.
I look forward to trying it out again in the future.

**Reply by jsmorabito** [reply]
no worries, thanks for the feedback!
